tests/unit/dom_exception_test.ts

```diff
@@ -30,3 +30,39 @@ Deno.test(function hasStackAccessor() {
   assert(typeof desc.get === "function");
   assert(typeof desc.set === "function");
 });
+
+Deno.test(function quotaExceededErrorIsSubclass() {
+  const error = new QuotaExceededError("test message");
+  assert(error instanceof QuotaExceededError);
+  assert(error instanceof DOMException);
+  assert(error instanceof Error);
+});
+
+Deno.test(function quotaExceededErrorCodeIsZero() {
+  // QuotaExceededError is now a subclass, not a DOMException name
+  // So creating a DOMException with name "QuotaExceededError" should have code 0
+  const error = new DOMException("test", "QuotaExceededError");
+  assertEquals(error.code, 0);
```

Comment:
The WHATWG PR comment states the following, is this correct?

* `DOMException.QUOTA_EXCEEDED_ERR === 22`
* `(new DOMException("message", "QuotaExceededError")).code === DOMException.QUOTA_EXCEEDED_ERR`